tests/system/src/test/java/test/robot/javafx/scene/MouseLocationOnScreenTest.java

```diff
@@ -131,13 +147,23 @@ private static void cross(Robot robot, int x0, int y0, int x1, int y1) {
         double dy = (y1 - y0) / dmax;
 
         robot.mouseMove(x0, y0);
+        delay(delayTime);
         validate(robot, x0, y0);
 
         for (int i = 1; i <= dmax; i++) {
             int x = (int) (x0 + dx * i);
             int y = (int) (y0 + dy * i);
             robot.mouseMove(x, y);
+            delay(delayTime);
             validate(robot, x, y);
         }
     }
+
+    private static void delay(int time) {
+        try {
+            Thread.sleep(time);
+        } catch (InterruptedException e) {
+
```

Comment:
Minor: there is an extra blank line here, although if you use `Util.sleep` instead you can remove this method entirely.